### DISCUSSION TITLE
Parameters as environment variables validation

### DISCUSSION THREAD

**Discussion by rmikalkenas** [symfony/symfony#47508] [Q&A] [answered]
At the moment parameters which hold environment variables are resolved during runtime. Is there a way or ideally a command which would validate if all env variables are resolvable/defined?

An issue I'm trying to solve is to reduce the possibility (by adding a CI check) to miss value of env variable in .env and release such code to production

**Comment by MatTheCat** [comment, marked as answer]
Does `debug:container --env-vars` address your use-case?
